#!/bin/sh /usr/share/dpatch/dpatch-run
## 03-config-kmk.dpatch by Patrick Winnertz <patrick.winnertz@skolelinux.org>
##
## DP: Some patches for Config.kmk to build with the debian tools
## DP: XPCOM setup for all but win32.

@DPATCH@

--- virtualbox-ose-1.6.0-dfsg/Config.kmk.orig	2008-05-01 18:24:58.000000000 +0200
+++ virtualbox-ose-1.6.0-dfsg/Config.kmk	2008-05-01 18:25:43.000000000 +0200
@@ -970,7 +970,7 @@
 # We need to export YASM for OSE, but since it was converted to .zip we need to
 # allow for the location used by the zip to be properly cleaned up first.
 if1of ($(BUILD_PLATFORM).$(BUILD_PLATFORM_ARCH),  win.x86 linux.x86 solaris.x86)
- PATH_TOOL_YASM ?= $(PATH_DEVTOOLS_BLD)/bin
+ PATH_TOOL_YASM ?= /usr/bin
 endif
 
 # XPCOM setup for all but win32.
@@ -1456,7 +1456,7 @@
    kernelsrc := $(shell $(SED_EXT) -ne 's+^KERNELSRC.*= *\(.*\)+\1+p' $(VBOX_LINUX_SRC)/Makefile)
    ifneq ($(kernelsrc),)
     kernelpath := $(realpath $(VBOX_LINUX_SRC)/$(kernelsrc))
-    VBOX_LINUX_INCS                 += $(VBOX_LINUX_SRC)/include2 $(kernelpath)/include
+    VBOX_LINUX_INCS                 += $(VBOX_LINUX_SRC)/include2 /usr/include
    else
     kernelpath := $(VBOX_LINUX_SRC)
    endif
